packages/ember-routing/lib/system/router.js

```diff
@@ -705,6 +706,24 @@ var EmberRouter = EmberObject.extend(Evented, {
       run.cancel(this._slowTransitionTimer);
     }
     this._slowTransitionTimer = null;
+  },
+
+  _handledErrors: computed(function() {
```

Comment:
That was my first attempt, but the problem I hit was that there's no insurance that the error isn't a primitive type, and there are several tests where it IS a primitive type. This wouldn't be a difficult change (to fire an assertion of the type isn't an object, and update the tests) but it would probably be a problem for a lot of apps. It would need to be noted/documented/blogged about or something.